.idea/workspace.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="AutoImportSettings">
+    <option name="autoReloadType" value="SELECTIVE" />
+  </component>
+  <component name="ChangeListManager">
```

Comment:
`git rm .idea`

do not commit generated files

Reply:
you have 65 changed files and most of them were generated by IDE or compiler. please celanup
https://github.com/jarviscanada/jarvis_data_eng_EkeneAnonyai/pull/7/files 

 open another PR review on Notion please